Client/Ecosia/UI/NTP/BookmarkNudge/NTPBookmarkNudgeCellViewModel.swift

```diff
@@ -58,7 +58,7 @@ extension NTPBookmarkNudgeCellViewModel: HomepageViewModelProtocol {
     }
 
     var isEnabled: Bool {
-        User.shared.showsBookmarksNTPNudgeCard()
+        User.shared.showsBookmarksNTPNudgeCard() // TODO: Check why this logic is broken (`firstTime` false the first time it's executed)
```

Comment:
I believe this might be related with the change asynchronous code to present onboarding, but bookmarks nudge card logic is broken since when it gets here the first time `firstTime` is already false for the user.

Reply:
The issue here is not encountered, especially after the latest changes performed.